examples/auth_example/auth_example_client/lib/src/protocol/protocol.dart

```diff
@@ -41,27 +41,29 @@ class Protocol extends _i1.SerializationManager {
   }
 
   @override
-  String? getClassNameForObject(Object data) {
+  String? getClassNameForObject(Object? data) {
+    String? dartTypeClassName = super.getClassNameForObject(data);
```

Comment:
Suggestion: Combine this with the `className` variable.

This variable serves the same purpose as the `className` variable bellow, might as well reuse it for the same purpose.